CNAME

```diff
@@ -1 +0,0 @@
-fossasia.org
```

Comment:
why deleting this?

Reply:
I apologize for this, but GitHub was showing me a problem while deploying my branch due to same CNAME.
So, I temporarily deleted that file. If you are satisfied with the change made in the MEMBERSHIP text, then I can add the CNAME file again.

Reply:
What kind of problem? Add the file and use another method for deployment like netlify. Also after that, squash the commit.

Reply:
Please don't remove cname, it's required for deployment via GitHub pages and custom domain